README.md

```diff
@@ -39,10 +39,11 @@ Here are all the libraries available in the group `com.google.android.filament`:
 
 - `filament-android`: the Filament rendering engine itself
 - `gltfio-android`: a glTF 2.0 loader for Filament, depends on `filament-android`
+- `filament-utils-android`: KTX loading, Kotlin math, and camera utilities, depends on `gltfio-android`
 - `filamat-android-full`: a runtime material builder/compiler. This library is large but contains
-  a full shader compiler/validator/optimizer
+  a full shader compiler/validator/optimizer.
```

Comment:
You added a period here but the other items above don't have a period at the end :)